pkg/asset/manifests/clusterapi/cluster.go

```diff
@@ -42,7 +43,8 @@ var _ asset.WritableRuntimeAsset = (*Cluster)(nil)
 // Cluster generates manifests for target cluster
 // creation using CAPI.
 type Cluster struct {
-	FileList []*asset.RuntimeFile
+	FileList       []*asset.RuntimeFile
+	CapiClusterIDs []string
```

Comment:
Bikeshedding here, I don't really like the `CapiClusterIDs` in the name. For sure we can drop the `Capi` prefix?

Reply:
Sure.  I can change that name.  I only put capi to let it be known its the ID of the cluster in CAPI during install.  Its normally the ocp cluster name + #.